test/unit/rules/no-aria-hidden-body-test.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const { ERROR_MESSAGE } = require('../../../lib/rules/no-aria-hidden-body');
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'no-aria-hidden-body',
+
+  config: true,
+
+  good: ['<body></body>', '<body><h1>Hello world</h1></body>'],
```

Comment:
You need to make sure `aria-hidden` is allowed on non-body elements.

Reply:
done

Reply:
thank you!